src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1722,6 +1758,7 @@ def __call__(
                 token_type_ids=jnp.array(token_type_ids, dtype="i4"),
                 position_ids=jnp.array(position_ids, dtype="i4"),
                 head_mask=jnp.array(head_mask, dtype="i4"),
+                indices_prng_key=jnp.array(indices_prng_key, dtype="u4"),
```

Comment:
We shouldn't pass the indices PRNG key as an arg, but rather in the dict of RNGs (see above)

```suggestion
```